jdks/download-jdks.sh

```diff
@@ -89,21 +90,25 @@ function unpack_mac_jdk {
         xar -xf /Volumes/JDK*/JDK*.pkg
         hdiutil detach /Volumes/JDK*
     else # Linux
-        7z x ../downloads/jdk-macosx-x64.dmg > /dev/null
+        7z x ../downloads/jdk-macosx-x64.dmg 
         # The following seems dependent of the 7zip version. If not they also changed their MacOSX Installer
-        7z x 4.hfs > /dev/null
-        install_xar
-        ./xar-1.5.2/src/xar -xf JDK*/JDK*.pkg
+        #7z x 4.hfs > /dev/null
+        #install_xar
+        #./xar-1.5.2/src/xar -xf JDK*/JDK*.pkg
+    cd JDK*
```

Comment:
In theory this could also be done like `7z x JDK*/JDK*.pkg`, however I guess both ways are fine here. I just wanted to note that for the sake of completeness